### PR TITLE
PA-792: Fix `credName` issue in `IssueDeleteOfIncrementalBackupsAndRestore`

### PR DESCRIPTION
**Pull request by kshithijiyer-px**
**What this PR does / why we need it**:
- [x] Fix `credName` issue in `IssueDeleteOfIncrementalBackupsAndRestore` created due to PR https://github.com/portworx/torpedo/pull/1228
- [x] Fix `cloudCredUID` issue in `IssueDeleteOfIncrementalBackupsAndRestore` created due to PR https://github.com/portworx/torpedo/pull/1228
- [x] Add logic to wait for cloud cred to be created before going to create backup location. 
- [x] Fix typo in comment 

Without fix: http://aetos.pwx.purestorage.com/resultSet/testSetID/138404/testCaseID/703611
With fix:  http://aetos.pwx.purestorage.com/resultSet/testSetID/142512

**Which issue(s) this PR fixes** (optional)
Closes # PA-792

**Special notes for your reviewer**:
Will squash and merge.

### CODE REVIEW

File: tests/backup/backup_delete_test.go [hunk 1]
```diff
@@ -74,9 +75,20 @@ var _ = Describe("{IssueDeleteOfIncrementalBackupsAndRestore}", func() {
...
-				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of cloud credential named [%s] for org [%s] with [%s] as provider", CredName, orgID, provider))
+				err := CreateCloudCredential(provider, credName, cloudCredUID, orgID, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of cloud credential named [%s] for org [%s] with [%s] as provider", credName, orgID, provider))
 				log.InfoD("Created Cloud Credentials with name - %s", credName)
+				cloudCredCreateStatus := func() (interface{}, bool, error) {
+					status, err := IsCloudCredPresent(credName, ctx, orgID)
+					if err != nil {
+						return "", true, fmt.Errorf("cloud cred %s is not created with error %v", credName, err)
+					}
+					if status {
```
Comment (tthurlapati-px) [inline tests/backup/backup_delete_test.go:86]: nit: Variable name `status` is not representation of something that is boolean; 
maybe use `ok` or `isPresent` or `cloudCredPresent`.

**Review by tthurlapati-px — COMMENTED** [review]
Other than the following, LGTM

**Review by tthurlapati-px — APPROVED** [review]
LGTM

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Pull request merged**